src/main/java/com/zotov/edu/passportofficerestservice/converter/PersonDtoConverter.java

```diff
@@ -0,0 +1,54 @@
+package com.zotov.edu.passportofficerestservice.converter;
+
+import com.zotov.edu.passportofficerestservice.model.entity.Person;
+import com.zotov.edu.passportofficerestservice.model.request.PersonRequest;
+import com.zotov.edu.passportofficerestservice.model.response.PersonResponse;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Component
+public class PersonDtoConverter implements PersonConverter {
+
+    @Override
+    public Page<PersonResponse> convertToDto(Page<Person> personPage) {
+        List<PersonResponse> personResponses = convertToDto(personPage.getContent());
+        return new PageImpl<>(personResponses);
+    }
+
+    @Override
+    public PersonResponse convertToDto(Person person) {
+        return new PersonResponse(
+                person.getId(),
+                person.getName(),
+                person.getBirthday(),
+                person.getCountry());
+    }
+
+    @Override
+    public List<PersonResponse> convertToDto(List<Person> persons) {
+        return persons.stream().map(this::convertToDto).collect(Collectors.toList());
+    }
+
+    @Override
+    public Person convertToEntity(PersonRequest personRequest) {
```

Comment:
Currently `PersonRequest` - a DTO from controller level - is used in service level, that's not nice from onion arch perspective :) where each level can work with DTOs of its own and lower one.

Since we don't have many fields in request, we can pass them to service just in method parameters.

Reply:
The service methods input args have been changed to not use controller-related DTOs.